resources/config.json

```diff
@@ -55,6 +55,32 @@
                     "git://github.com/ugexe/Perl6-ecosystems.git",
                     "http://ecosystem-api.p6c.org/projects1.json"
                 ]
+            }
+				},
+        {
+            "short-name": "zef-p6c",
+            "enabled": 1,
+            "module": "Zef::Repository::Ecosystems",
+            "options": {
+                "name": "zef-p6c",
+                "auto-update": 1,
+                "uses-path": true,
+                "mirrors": [
+                    "http://32.zef.pm/"
+                ]
+            }
+        },
+        {
+            "short-name": "zef",
```

Comment:
This name isn't good because it means to disable it or enable it reads `zef install Foo::Bar --/zef` (a `--zef` or `--/zef` is too ambiguous)